skbio/tree/tests/test_tree.py

```diff
@@ -829,6 +829,11 @@ def test_tips(self):
         obs2 = [n.name for n in self.simple_t.traverse(False, False)]
         self.assertEqual(obs2, exp)
 
+    def test_tips_self(self):
+        tree = TreeNode.read(['(c, (b,a)x)y;'])
+        t = next(tree.children[0].tips(include_self=True))
+        self.assertEqual(t.name, 'c')
```

Comment:
Please assert it is a tip as well. Actually, I think the test can be made more succinct with: `self.assertEqual(list(tree.find('c').tips(include_self=True)), [tree.find('c')])